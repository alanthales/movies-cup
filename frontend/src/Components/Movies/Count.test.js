import React from "react";
import MoviesCount from "./Count";
import renderer from "react-test-renderer";

test("render Movies Count from given props", () => {
  const component = renderer.create(
    <MoviesCount value={4} onGenerate={() => {}} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});