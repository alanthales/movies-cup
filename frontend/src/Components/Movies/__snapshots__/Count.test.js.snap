// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Movies Count from given props 1`] = `
<div
  className="MuiGrid-root movies-component-count MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <h4>
      Selecionados
    </h4>
    <p>
      4 de 8
    </p>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeLarge"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Gerar Meu Campeonato
      </span>
    </button>
  </div>
</div>
`;
